### PR TITLE
4 banner mission vision join us

### PR DESCRIPTION
**Pull request by DeborahLacerda**
I did, but I have some issues with the title - On figma  at normal size **Our mission** and **Our vision** is font size: 50px and **Join us** one font size: 48px In responsive "small" Both have equal font size: 40px; Then assume the titles are equal, if it is different I can make other components.

Another thing is One inconsistency in the paragraph/description (28px) with the figma caption ( body/b1) I choose body/b1. 
Any problem let me know.

### CODE REVIEW

File: components/Banner/Banner.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import stylesbanner from '@/styles/banner.module.css'
```
Comment (marceloglacial) [inline components/Banner/Banner.jsx:1]: Please close this PR and open another one pointing to develop branch

**Review by marceloglacial — CHANGES_REQUESTED** [review]

**Comment by marceloglacial** [comment]
> I did, but I have some issues with the title - On figma at normal size **Our mission** and **Our vision** is font size: 50px and **Join us** one font size: 48px In responsive "small" Both have equal font size: 40px; Then assume the titles are equal, if it is different I can make other components.
> 
> Another thing is One inconsistency in the paragraph/description (28px) with the figma caption ( body/b1) I choose body/b1. Any problem let me know.

No worries. Let's keep the way you did and ask the question on the [#ui-issues thread](https://discord.com/channels/887715490183667723/1080554768004550697).

File: components/Banner/Banner.jsx (hunk 1 above)
Comment (marceloglacial) [reply components/Banner/Banner.jsx:1]: TY

File: components/Banner/Banner.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+
+
+const Banner = (props) => {
```
Comment (marceloglacial) [inline components/Banner/Banner.jsx:5]: Great job!

File: components/Banner/Banner.jsx (hunk 1 above)
Comment (marceloglacial) [inline components/Banner/Banner.jsx:1]: Please use the patter `styles` and the initial capital letter for the css file

File: components/Banner/index.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import Head from 'next/head';
```
Comment (marceloglacial) [inline components/Banner/index.jsx:1]: This file is no needed. You can remove it.

File: constants/labels.en.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+const labels = {
```
Comment (marceloglacial) [inline constants/labels.en.js:1]: Great idea

File: pages/index.jsx [hunk 5]
```diff
@@ -1,6 +1,8 @@
 import Head from 'next/head';
```
Comment (marceloglacial) [inline pages/index.jsx:1]: There is an updated index file on `develop` branch. Please merge `develop` into your branch before create the PR.

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: components/Banner/Banner.jsx (hunk 1 above)
Comment (DeborahLacerda) [reply components/Banner/Banner.jsx]: Done!!

File: components/Banner/index.jsx (hunk 3 above)
Comment (DeborahLacerda) [reply components/Banner/index.jsx]: remove

File: pages/index.jsx (hunk 5 above)
Comment (DeborahLacerda) [reply pages/index.jsx:1]: Done

File: components/Banner/Banner.jsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import stylesbanner from '@/styles/Banner.module.css'
```
Comment (marceloglacial) [inline components/Banner/Banner.jsx:1]: Please use the pattern `styles` instead of `stylesbanner`

File: components/Banner/index.jsx [hunk 7]
```diff
@@ -0,0 +1,42 @@
+import { AppLayout } from '@/components';
```
Comment (marceloglacial) [inline components/Banner/index.jsx:1]: You can delete the file itself. It's no. needed.

File: components/Banner/Banner.jsx (hunk 6 above)
Comment (marceloglacial) [reply components/Banner/Banner.jsx:1]: please remove the empty lines

File: pages/index.jsx [hunk 8]
```diff
@@ -14,13 +16,14 @@ export default function Home() {
...
-        <Placeholder title='Our Mission' />
-        <Placeholder title='Our Vision' />
+        <Banner title={labels.ourMission.title} description={labels.ourMission.description}/>
+        <Banner title={labels.ourVision.title} description={labels.ourVision.description}/>
         <Placeholder title='Our Board' />
-        <Placeholder title='Join Us' />
+        <Banner title={labels.joinUs.title} description={labels.joinUs.description}/>
         <Placeholder title='Form' />
         <Placeholder title='Contact Us' />
+
```
Comment (marceloglacial) [inline pages/index.jsx:25]: please remove the empty line

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: components/Banner/Banner.jsx (hunk 6 above)
Comment (DeborahLacerda) [reply components/Banner/Banner.jsx]: done

File: components/Banner/index.jsx (hunk 7 above)
Comment (DeborahLacerda) [reply components/Banner/index.jsx]: done

File: pages/index.jsx (hunk 8 above)
Comment (DeborahLacerda) [reply pages/index.jsx]: done

**Review by marceloglacial — APPROVED** [review]

**Pull request merged**